server/controllers/viewMentor.js

```diff
@@ -20,5 +20,25 @@ getAll = async (req, res) => {
     });
   }
 }
+getOne = async (req, res) => {
+  try {
+    const newMentor = await allMentors.getOneService(req.params.id);
+    if (newMentor == undefined) {
```

Comment:
Use `!newMentor` to check if the value is false rather than checking its equality to undefined. 

The value could be null instead of undefined